src/homework/forth/MyArrayList.java

```diff
@@ -0,0 +1,53 @@
+package homework.forth;
+
+public class MyArrayList<T> implements List<T> {
+    private Object[] arr;
+    private int counter;
+
+    public MyArrayList() {
+        arr = new Object[10];
+    }
+
+    public T get(int index) {
+        if (index < 0 || index >= counter) {
+            System.out.println("Error. Please check the index " + index);
```

Comment:
Create your own class exception and it's an object instead of return null. 
As the element of the list could be null. When we get null we are not able to say is it an element or not right index.